RocketControlUnitGUI/src/routes/+page.svelte

```diff
@@ -410,6 +410,43 @@
 		});
 	}
 
+	let wasLiveAtAnyPoint = false;
+
+	async function pollIgnitors() {
+		if (box1_display === 'LIVE' || box2_display === 'LIVE') {
+			return wasLiveAtAnyPoint = true;
+		}
+
+		return wasLiveAtAnyPoint;
+	}
+
+	async function handleLaunchSequence() {
+		await PB.collection('CommandMessage').create({
+			target: 'NODE_RCU',
+			command: 'RCU_IGNITE_PAD_BOX1'
+		});
+
+		await PB.collection('CommandMessage').create({
+			target: 'NODE_RCU',
+			command: 'RCU_IGNITE_PAD_BOX2'
+		});
+
+		const pollInterval = setInterval(pollIgnitors, 100);
+
+		await new Promise(resolve => setTimeout(resolve, 3500));
+
+		clearInterval(pollInterval);
+
+		if (wasLiveAtAnyPoint) {
+			await PB.collection('CommandMessage').create({
+				target: 'NODE_DMB',
+				command: 'RSC_IGNITION_TO_LAUNCH'
+			});
```

Comment:
should we do the thing to send 3x with like 100ms delay?

Reply:
done